plugins/c8y_log_plugin/src/main.rs

```diff
@@ -225,17 +225,18 @@ fn create_init_logs_directories_and_files(
 
     // creating c8y-log-plugin.toml
     let logs_path = format!("{logs_dir}/tedge/agent/software-*");
-    let data = toml::toml! {
-        files = [
-            { type = "software-management", path = logs_path }
-        ]
-    };
+    let data = format!(
+        "files = [
+    {{ type = \"software-management\", path = \"{logs_path}\" }},
+]"
+    );
```

Comment:
Yes, thats right. It's because of `toml::toml!`. Comment in the PR description you mean?

Reply:
> Please use a raw string `r#" ... "` to avoid all these escape sequences.

Using `r#` only removes the `\` escapes not `{`. Still it's a good idea.